packages/flame_forge2d/pubspec.yaml

```diff
@@ -4,17 +4,20 @@ version: 0.11.0
 homepage: https://github.com/flame-engine/flame/tree/main/packages/flame_forge2d
 
 environment:
-  sdk: ">=2.16.0 <3.0.0"
+  sdk: ">=2.17.0 <3.0.0"
```

Comment:
should we keep this consistent across projects (ie 16)?
if we want to update we should make a separate PR updating every package

Reply:
For sure! When do we plan to update the SDK? Is there something stopping us from it?

Reply:
I'm going to update after the release

Reply:
And it is consistent on all other packages, if that was a question :)